Sources/Core/Glob.swift

```diff
@@ -176,13 +176,13 @@ public class Glob: Collection {
                 if blacklistedDirectories.contains(subpath) {
                     return nil
                 }
-                let firstLevelPath = NSString(string: path).appendingPathComponent(subpath)
+                let firstLevelPath = NSString(string: path).appendingPathComponent(subpath).standardizingPath
                 guard isDirectory(path: firstLevelPath) else {
                     return nil
                 }
                 var subDirs: [String] = try FileManager.default.subpathsOfDirectory(atPath: firstLevelPath)
                     .compactMap { subpath -> String? in
-                        let fullPath = NSString(string: firstLevelPath).appendingPathComponent(subpath)
+                        let fullPath = NSString(string: firstLevelPath).appendingPathComponent(subpath).standardizingPath
```

Comment:
It looks like this is unavailable on macOS from the tests? `Foundation.StringProtocol:31:16: note: 'standardizingPath' has been explicitly marked unavailable here`

Reply:
Yeah – seeing the same thing here. It might make sense to move some of the internal here over to URL to avoid Linux/Mac Foundation incompatibilities.